src/puppetlabs/puppetdb/mq.clj

```diff
@@ -17,7 +17,8 @@
 (def mq-metrics-registry (get-in metrics/metrics-registries [:mq :registry]))
 
 (def metrics (atom {:message-persistence-time (timer mq-metrics-registry
-                                                     (metrics/keyword->metric-name [:global] :message-persistence-time))}))
+                                                     (metrics/keyword->metric-name
```

Comment:
I think we've got some general purging of MQ related dead code. My guess is this will go when that does.